packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/generalized-react-app.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Template file for new grouping rules. To use:
+ *
+ * 1. Copy TEMPLATE.js to a new grouping rule file.
+ * 2. Change `ChangeMeGroupingRule` class name to another name.
+ * 3. Change `this.name` from `CHANGE_ME` in the constructor.
+ * 4. Fill in the `choose` function.
+ * 5. Include the new rule file in index.js.
+ * 6. Access in views via `this.experiments.choose('name from 3')`
+ *    or `this.isInExperimentGroup('name from 3', 'group name')`.
+ */
+ 'use strict';
+
+ const BaseGroupingRule = require('./base');
+
+ const GROUPS = [
+   'control',
+
+   // Treatment branches. This one is for users who will see the new, generalized React app which houses more urls than just `/settings`
+   'generalized',
+ ];
+
+ // For each client specify which experiment group to show
+ const ROLLOUT_CONFIG = {
+   // 123Done
+   dcdb5ae7add825d2: GROUPS,
+   // Pocket
```

Comment:
Not sure we need this for Pocket especially 🤔 What's this do?

Reply:
This gives us the ability to open roll out changes to certain clients (as I understand it) but I think you're right -- at least for this experiment, we probably don't need it. I'll remove!